TombEngine/Game/control/box.cpp

```diff
@@ -1194,37 +1190,21 @@ int CreatureVault(short itemNumber, short angle, int vault, int shift)
 
 	// FIXME: Add climb down animations for Von Croy and baddies?
 	if (item->Floor > y + CLICK(4.5f))
-	{
 		vault = 0;
-	}
-	else if (item->Floor > (y + CLICK(3.5f)) && IsCreatureVaultAvailable(item, -4))
-	{
+	else if (item->Floor > y + CLICK(3.5f))
 		vault = -4;
-	}
-	else if (item->Floor > (y + CLICK(2.5f)) && IsCreatureVaultAvailable(item, -3))
-	{
+	else if (item->Floor > y + CLICK(2.5f) && IsCreatureVaultAvailable(item, -3))
 		vault = -3;
-	}
-	else if (item->Floor > (y + CLICK(1.5f)) && IsCreatureVaultAvailable(item, -2))
-	{
+	else if (item->Floor > y + CLICK(1.5f) && IsCreatureVaultAvailable(item, -2))
 		vault = -2;
-	}
-	else if (item->Pose.Position.y > (y - CLICK(1.5f)))
-	{
+	else if (item->Pose.Position.y > y - CLICK(1.5f))
 		return 0;
-	}
-	else if (item->Pose.Position.y > (y - CLICK(2.5f)))
-	{
+	else if (item->Pose.Position.y > y - CLICK(2.5f))
 		vault = 2;
-	}
-	else if (item->Pose.Position.y > (y - CLICK(3.5f)))
-	{
+	else if (item->Pose.Position.y > y - CLICK(3.5f))
 		vault = 3;
-	}
-	else if (item->Pose.Position.y > (y - CLICK(4.5f)))
-	{
+	else
```

Comment:
Why you have removed `else if (item->Pose.Position.y > (y - CLICK(4.5f)))` condition here again, after I have added it back from tomb4?